components/portal/messages/controllers.js

```diff
@@ -175,6 +175,20 @@ define(['angular'], function(angular) {
         // Local methods //
         // ////////////////
         /**
+         * Check if user is viewing notifications page
+         */
+        var isNotificationsPage = function() {
+          if (MESSAGES.notificationsPageURL.indexOf('/web') != -1) {
```

Comment:
:art: prefer strict equality check.

:thought_balloon: I have a dream that one day `Array.includes` will be supported by the browser matrix.